hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/PrefixParser.java

```diff
@@ -148,6 +148,14 @@ public void parse(String vol, String buck, String db,
       return;
     }
 
+    BucketLayout bucketLayout = info.getBucketLayout();
+    if (!bucketLayout.isFileSystemOptimized()) {
+      System.out.println("Prefix tool only works for FileSystem Optimized" +
+              "bucket. Bucket Type is:" + bucketLayout);
```

Comment:
@mukul1987 can you please modify message `"Bucket Type is:"` -> to -> `"Bucket Layout is:"`